topics/go/language/struct_types/example3/example3.go

```diff
@@ -39,7 +39,10 @@ func main() {
 	// Display the values.
 	fmt.Printf("%+v\n", ex)
 	fmt.Printf("%+v\n", e)
-	fmt.Println("Flag", e.flag)
-	fmt.Println("Counter", e.counter)
-	fmt.Println("Pi", e.pi)
+	fmt.Printf(
+		"Flag[%t] Counter[%d] Pi[%.6f]\n",
+		e.flag,
```

Comment:
Please revert this one. The point in this example is learning how to get/set struct field values using the `.` operator. I think students will be distracted by the printf verbs.

Reply:
Agree